addons/hr_recruitment/views/hr_job_views.xml

```diff
@@ -294,13 +294,23 @@
                 <field name="user_id" widget="many2one_avatar_user" optional="hide"/>
                 <field name="no_of_hired_employee" optional="hide"/>
             </field>
+            <tree position="attributes">
+                <attribute name="js_class">recruitment_list_view</attribute>
+            </tree>
         </field>
     </record>
 
     <!-- hr related job position menu action -->
+    <record id="action_load_demo_data" model="ir.actions.server">
+        <field name="name">Load demo data</field>
+        <field name="model_id" ref="hr_recruitment.model_hr_job"/>
+        <field name="state">code</field>
+        <field name="code">action = model._action_load_recruitment_scenario()</field>
+    </record>
+
     <record model="ir.actions.act_window" id="action_hr_job">
         <field name="name">Job Positions</field>
-        <field name="path">recruitement</field>
+        <field name="path">recruitment</field>
```

Comment:
don't forget to ask someone about this changes

Reply:
@jpp-odoo do you have a solution to fix this kind of typo in paths?

Reply:
@KangOl The idea was if the path needs to be changed (as is the case here because of a typo), a script migration can be created.
The test was added to avoid accidental modifications. The test also avoids changing a path that the user manually modified (by the window action interface).

Reply:
That doesn't resolve the fact that it break the shared URLs. I thought that was why you wanted to be blocked at upgrade.

Reply:
Hello @KangOl Indeed, we will lose the retro-compatibility but it occurs after each upgrade (action does not exist anymore or whatever). So, not a big deal.

CC: @jpp-odoo